<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>906</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 28pt &quot;Segoe UI&quot;; color: #4CAF50;</string>
   </property>
   <property name="text">
    <string>USERNAME:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameInput">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;MS Shell Dlg 2&quot;; padding: 10px; border-radius: 10px; border: 2px solid #4CAF50;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 28pt &quot;MS Shell Dlg 2&quot;; color: #4CAF50;</string>
   </property>
   <property name="text">
    <string>Equation:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="equaInput">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>631</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;; padding: 10px; border-radius: 10px; border: 2px solid #4CAF50;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>360</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 24pt &quot;Segoe UI Semilight&quot;; color: #4CAF50;</string>
   </property>
   <property name="text">
    <string>Answer here</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="anwserInput">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>430</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;MS Shell Dlg 2&quot;; padding: 10px; border-radius: 10px; border: 2px solid #4CAF50;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>530</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 18pt &quot;Segoe UI&quot;; background-color: #4CAF50; color: white; border-radius: 10px; padding: 10px;</string>
   </property>
   <property name="text">
    <string>SUBMIT</string>
   </property>
  </widget>
  <widget class="QLabel" name="valArea">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>600</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;; color: #4CAF50;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="resArea">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>650</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;; color: #4CAF50;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>750</y>
     <width>1161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="playAgain">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>790</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 18pt &quot;Segoe UI&quot;; background-color: #2196F3; color: white; border-radius: 10px; padding: 10px;</string>
   </property>
   <property name="text">
    <string>Play Again</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>790</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 18pt &quot;Segoe UI&quot;; background-color: #F44336; color: white; border-radius: 10px; padding: 10px;</string>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>1121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
